ui/main/src/app/services/admin/SupervisedEntitiesService.ts

```diff
@@ -30,7 +31,12 @@ export class SupervisedEntitiesService {
                 if (entitiesResponse.status === ServerResponseStatus.OK) {
                     SupervisedEntitiesService.deleteFromCachedEntities(id);
                 } else {
-                    ErrorService.handleServerResponseError(entitiesResponse);
+                    logger.error(`Error while deleting entity :  ${entitiesResponse.statusMessage}`);
```

Comment:
Maybe we should add in the log the id of the entity we're trying to delete ?

Reply:
ok